cli/printers/text_printer.py

```diff
@@ -48,62 +56,41 @@ def _print_detection_summary(self, detection: Detection, document_path: str, sca
             f'(rule ID: {detection.detection_rule_id}) in file: {click.format_filename(document_path)} ' +
             f'{detection_sha_message}{detection_commit_id_message}  ⛔ ')
 
-    def _print_detection_code_segment(self, detection: Detection, document: Document, scan_type: str,
-                                      show_secret: bool, code_segment_size: int):
-        detection_details = detection.detection_details
-        detection_line = detection_details.get('line', -1) if scan_type == SECRET_SCAN_TYPE else \
-            detection_details.get('line_in_file', -1)
-        detection_position = detection_details.get('start_position', -1)
-        violation_length = detection_details.get('length', -1)
-
-        document_content = document.content
-        document_lines = document_content.splitlines()
-        start_line = self._get_code_segment_start_line(detection_line, code_segment_size)
-        detection_position_in_line = self._get_position_in_line(document_content, detection_position)
-
-        click.echo()
-        for i in range(code_segment_size):
-            current_line_index = start_line + i
-            if current_line_index >= len(document_lines):
-                break
+    def _print_detection_code_segment(self, detection: Detection, document: Document, code_segment_size: int):
+        if document.is_git_diff_format:
```

Comment:
@artem-fedorov I don't understand. what I need to fix? 😅

Reply:
Look like it is just a warning, because `is_git_diff_format` can be a function and you should use `. is_git_diff_format()`, because `.is_git_diff_format` will always return true

Reply:
https://semgrep.dev/r?q=%24FUNC%22+a+function+or+an+attribute%3F+If+it+is+a+f

Reply:
#cycode_sast_ignore_here
--


<br class="Apple-interchange-newline">